plaso/engine/processing_status.py

```diff
@@ -522,12 +522,20 @@ class EventsStatus(object):
   """The status of the events.
 
   Attributes:
+    number_of_duplicate_events (int): number of duplicate events.
```

Comment:
Please add more verbose descriptions of what the counters are. Specifically:
- Do the duplicate events just count events beyond the first one? ie, if events A and B are duplicates, is this number 1 or 2? If A, B and C are duplicates, is this number 1, 2 or 3?
- Is the number of filtered events the number filtered out of the output, or the number filtered into the output?
- Is the number of MACB group events the number of events grouped together, or number of coalesced events? ie, if an Access and Modify events are grouped together, is this number 1 or 2?

Reply:
> Do the duplicate events just count events beyond the first one?

Yes

> ie, if events A and B are duplicates, is this number 1 or 2?

it contains 1, since A is seen as the original

> If A, B and C are duplicates, is this number 1, 2 or 3?

it contains 2, since A is seen as the original

> Is the number of filtered events the number filtered out of the output, or the number filtered into the output?

This is number of events excluded by the event filter.

> Is the number of MACB group events the number of events grouped together, or number of coalesced events? ie, if an Access and Modify events are grouped together, is this number 1 or 2?

I assume with "coalesced events" you are referring to the resulting MACB group?
This value contains the number of events grouped, not the number of resulting groups.